Assets/HoloToolkit/UX/Scripts/Collections/CollectionEnums/OrientTypeEnum.cs

```diff
@@ -11,7 +14,11 @@ public enum OrientTypeEnum
         None,                   // Don't rotate at all
         FaceOrigin,             // Rotate towards the origin
         FaceOriginReversed,     // Rotate towards the origin + 180 degrees
-        FaceFoward,             // Zero rotation
-        FaceForwardReversed,    // Zero rotation + 180 degrees
```

Comment:
same comments as other PR about obsolete messages and not removing and just assigning them to the new enum values.